utilities/validations.py

```diff
@@ -0,0 +1,27 @@
+from api.location.models import CountryType, TimeZoneType
+
+# new file was created as imports are not possible in utility file
+
+
+def validate_country_field(**kwargs):
+    """
+    Function to validate country fields when
+    saving an object
+    :params kwargs
+    """
+    valid_countries = [country.value for country in CountryType]
+    country_name = kwargs['country']
+    if country_name not in valid_countries:
+        raise AttributeError("Not a valid country")
+
+
+def validate_timezone_field(**kwargs):
+    """
+    Function to validate country fields when
+    saving an object
```

Comment:
@bevkololi please correct the documentation here.  It should be to validate timezone fields, not country fields. 
Good work though.

Reply:
@kampkelly, thanks, correcting this now.